app/src/main/java/org/mozilla/focus/utils/DialogUtils.kt

```diff
@@ -468,7 +468,7 @@ object DialogUtils {
         ThemeSettingDialogBuilder(activity, homeViewModel).show()
     }
 
-    fun showSetAsDefaultBrowserDialog(activity: FragmentActivity) {
+    fun showSetAsDefaultBrowserDialog(activity: FragmentActivity, homeViewModel: HomeViewModel) {
```

Comment:
how about pass a callback instead of viewmodel?

Reply:
Fixed